### ISSUE TITLE
Improve display of spacebar-text

### ISSUE THREAD

**Issue by Glitchy-Tozier** [florisboard/florisboard#1067]
Motivation:
![Screenshot_20210714-210243_KISS launcher.jpg](https://user-images.githubusercontent.com/59611881/125679388-ba08ed90-9cbb-41c1-90ee-a291361fde10.jpg)

(Screenshot taken in v0.3.13-beta05)

Suggested fix: Make text scroll across spacebar.

**Comment by patrickgold** [comment]
I agree that the text is currently cut off, seems like a small issue in the font sizing calculation. The previous solution was and will be to make the font smaller in case there is not enough room, like GBoard does it as well.

**Comment by lm41** [comment]
This is currently implemented as elipsis (...) if it would overflow.

**Comment by lm41** [comment]
This is controllable with snyggv2.
